oncue-agent/src/main/java/oncue/agent/AbstractAgent.java

```diff
@@ -20,6 +20,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.function.Predicate;
```

Comment:
Apparently so, I had `$JAVA_HOME` set to the JDK1.8 home, and this compiled and ran without problems using `java` and `javac` from OpenJDK 1.7.

I'll change this so it actually compiles under 1.7.